src/components/filter/FilterMenu.js

```diff
@@ -26,41 +50,69 @@ export const FilterMenu = () => {
         <div className="categories-title">Categories</div>
         <div className="categories-body">
           <label className="categories-div">
-            <input type="checkbox" name="" id="" />
+            <input
+              type="checkbox"
+              name=""
+              id=""
+              checked={productState.showCategory.includes("Rasayana")}
+              onChange={() =>
+                productDispatch({ type: "CATEGORIZE", payload: "Rasayana" })
+              }
+            />
             Rasayana
           </label>
```

Comment:
this way is correct , just a suggestion but you can map over category array having all those categories that will remove the repetation of whole input html and their respective classes .